model/common/src/icon4py/model/common/math/helpers.py

```diff
@@ -98,7 +98,7 @@ def _grad_fd_tang(
     return grad_tang_psi_e
 
 
-@gtx.field_operator
+@gtx.field_operator(grid_type=gtx.GridType.UNSTRUCTURED)
```

Comment:
I ended up adding the `grid_type` while I was using field operators for the RBF computation but ended up removing it in the end. Technically this isn't needed anymore for this PR, but probably belongs here anyway. Keep?

Reply:
Question for Magda.

Reply:
I'll leave the grid types as it seemed a bit arbitrary where they were added and where not. I'll remove the added operators though.